client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -951,8 +957,12 @@ export function YourBrowserSupportToolkit() {
         useful tool if you get a bug report on your site from an unusual or
         particularly old browser that you do not have access to.
       </p>
+      <figure>
+        <img src="" width="" height="" alt="" />
```

Comment:
Image is here: https://drive.google.com/file/d/1r2E2o93Rz7NXqb-FgTzHEPBLDVXsRRo4/view?usp=sharing